ansible/roles/ocp4-workload-authentication/tasks/workload.yml

```diff
@@ -1,48 +1,64 @@
 ---
-# Implement your Workload deployment tasks here
-
-- name: Check that ocp4_idm_install is defined
+- name: Set up ocp4_workload_authentication combined dictionary
+  set_fact:
+    ocp4_workload_authentication: >-
+      {{ ocp4_workload_authentication_defaults
+       | combine(ocp4_workload_authentication_input  | default( {} ),
+                 ocp4_workload_authentication_secret | default( {} ))
+      }}
+- name: Print combined role variables
+  debug: var=ocp4_workload_authentication
```

Comment:
i would remove that debug or add `verbosity: 2` to it, since the secrets are merged into that variable too.

Reply:
Makes sense. Done.